packages/ember-glimmer/tests/integration/components/local-lookup-test.js

```diff
@@ -48,6 +48,20 @@ moduleFor('Components test: local lookup', class extends RenderingTest {
     this.assertText('Nested template says: Hi!', 'Re-render works');
   }
 
+  ['@htmlbars moduleName remains unchanged']() {
```

Comment:
Hmm. I'm happy to land this test here though too.  Can you remove the `@htmlbars` prefix, and open a separate PR against beta?